salt/modules/virt.py

```diff
@@ -1741,6 +1875,19 @@ def is_xen_hyper():
     return 'libvirtd' in __salt__['cmd.run'](__grains__['ps'])
 
 
+def is_bhyve_hyper():
```

Comment:
For more flexibility I would deprecate the `is_.*hyper` functions and replace them by a `hypervisor_type` one.